### PR TITLE
Change misleading statements in blog post, fixes #1790

### LINKED ISSUE #1790: Misleading statements in the blog post
Hey folks. This is regarding #1721.

Nice blog post. While this does not change the gist of the post, I found your explanation of the actual bug rather confusing.

First, the comparison of `tracestate` as multiple headers vs. `traceparent` is misleading. `tracestate` is a comma-separated list of entries, and since that's how RFC7230 defines header concatenation, it works as you described. But `traceparent` is not a list, it's a single unbreakable string, so the reasoning for header concatenation does not apply. Also your reasoning about Java vs. other SDKs seems backwards - one would argue it was Java SDK that was non-compliant with Trace-Context spec because it was supposed to fail to parse clearly malformed `traceparent`, not just take the first value.

Second, the screenshot shows that your `traceparent` has a list of a whole number of entries, each representing completely different traces. Where were they all coming from? Obviously not from the caller, the caller would only send one trace-id (unless the caller instrumentation was broken). I am guessing the issue may have been some static storage in nginx module where it was accumulating and adding all previously seen traceparent entries to each outbound request, instead of the single entry coming in the inbound request, but the discussion of the blog talks about different things.

Since this is a blog post on the official OTEL website, could you please correct these mis-statements?

cc @svrnm @sanketmehta28 @kpratyus

### PR DESCRIPTION
**Pull request by svrnm**
prettier introduced some more changes, but the relevant part is in line 210 onwards.

@yurishkuro does this sound good to you? TY

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
I like the changes. @yurishkuro?

File: content/en/blog/2022/debug-otel-with-otel/index.md [hunk 1]
```diff
@@ -208,15 +208,20 @@ backend:
...
+headers again and again, and since having multi value headers is covered by
+[RFC7230][], this didn't lead to an issue immediately.
 
-Doing some research on that topic, we found out, that this is coved by
-[RFC7230][] and is [allowed and defined][] for `tracestate`, but the [definition
-for `traceparent`][] does not provide details on that corner-case.
+We tested the capability to correlate from nginx to a downstream service with a
+Java application. And, without reading into the source code of the OTel Java
+SDK, it looks like that Java is flexible in taking a `traceparent` with multiple
+values. So propagation from nginx to the Java service worked, while in contract,
```
Comment (yurishkuro) [inline content/en/blog/2022/debug-otel-with-otel/index.md:218]:
```suggestion
values, even though such format is invalid per the W3C Trace Context specification. 
So propagation from nginx to the Java service worked, while in contrast,
```

File: content/en/blog/2022/debug-otel-with-otel/index.md [hunk 2]
```diff
@@ -208,15 +208,20 @@ backend:
...
+python (and other languages) do not provide that flexibility and propagation
```
Comment (yurishkuro) [inline content/en/blog/2022/debug-otel-with-otel/index.md:219]:
```suggestion
Python (and other languages) do not provide that flexibility and propagation
```

**Comment by svrnm** [comment]
thanks @yurishkuro , I committed your suggestions.

**Review by yurishkuro — APPROVED** [review]
thanks!

**Pull request merged**